config/prod/minidream-R-env-pub-prod-instance-ec2.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision an EC2 instance
```

Comment:
no. files in the PRs have changed.  you need to copy from the updated file, https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3/files

Reply:
Thanks @zaro0508, I think I used the updated template now.